pallets/security/src/default_weights.rs

```diff
@@ -0,0 +1,80 @@
+
+//! Autogenerated weights for security
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-11-18, STEPS: `100`, REPEAT: 10, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! HOSTNAME: `MacBook-Pro`, CPU: `<UNKNOWN>`
+//! EXECUTION: None, WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
+
+// Executed Command:
+// ./target/release/spacewalk-standalone
+// benchmark
+// pallet
+// --chain=dev
+// --pallet=security
+// --extrinsic=*
+// --steps=100
+// --repeat=10
+// --wasm-execution=compiled
+// --output=pallets/security/src/default_weights.rs
+// --template=.maintain/frame-weight-template.hbs
```

Comment:
This file is not actually autogenerated or is it? I thought you added everything manually? 😳